MathLib/Integration/GaussLegendre.h

```diff
@@ -29,6 +29,15 @@ struct GaussLegendre {
     static const double W[Order];
 };
 
+template <> double const GaussLegendre<1>::X[1];
```

Comment:
I see.
Just curious: Would it suffice to instantiate via:
`template<unsigned ORDER> double const GaussLegendre<ORDER>::X[ORDER];`